OracleWebLogic/samples/12212-msiserver/README.md

```diff
@@ -140,4 +142,65 @@ You can shutdown the service using
 
 `$ docker service rm city_activity_guide`
 
+Rolling update
+----------------------------------
+You can do rolling update of the service using the docker service update command.

```

Comment:
Hi Monica: I have added a description for the example based on your comment. Thanks.